src/main/webapp/app/iris/http-message.service.ts

```diff
@@ -44,6 +44,23 @@ export class IrisHttpMessageService {
             );
     }
 
+    /**
+     * resends a message for a session
+     * @param {number} sessionId
+     * @param {IrisClientMessage} message
+     * @return {Observable<EntityResponseType>}
+     */
+    resendMessage(sessionId: number, message: IrisClientMessage): Observable<EntityResponseType> {
+        message.messageDifferentiator = message.messageDifferentiator ?? Math.floor(Math.random() * this.MAX_INT_JAVA);
+        return this.httpClient.post<IrisServerMessage>(`${this.resourceUrl}/${sessionId}/messages/${message.id}/resend`, message, { observe: 'response' }).pipe(
```

Comment:
Do we need convert the `message.sentAt` and assign to the object here?

Reply:
No, since the server does not actually use the message that is send in the body. I therefore replaced in with undefined here. Good catch!